src/prayer/UserPrayerList/index.js

```diff
@@ -47,7 +47,11 @@ class UserPrayerList extends React.Component {
       <ModalView {...this.props} onClose={() => this.props.navigation.pop()}>
         <FlexedSafeAreaView forceInset={{ top: 'always' }}>
           <ScrollView>
-            <Query query={GET_USER_PRAYERS} fetchPolicy="cache-and-network">
+            <Query
+              query={GET_PRAYERS}
+              variables={{ type: 'USER' }}
+              fetchPolicy="cache-and-network"
+            >
               {({ loading, data: { userPrayers = [] } = {} }) => {
```

Comment:
`userPrayers` is going to have to be `prayers` here and below. Unless you want to rename it.